### PR TITLE
New package: remmina

### PR DESCRIPTION
**Pull request by cr6git**
Package request [#8134](https://github.com/voidlinux/void-packages/issues/8134).

### CODE REVIEW

File: srcpkgs/remmina/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'remmina'
+pkgname=remmina
+version=1.2.0rcgit.21
```
Comment (the-maldridge) [inline srcpkgs/remmina/template]: This is not a release version.  Please back up to the last stable release and update this PR.

File: srcpkgs/remmina/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+_gitver=1.2.0-rcgit.21
```
Comment (the-maldridge) [inline srcpkgs/remmina/template]: As general policy we do not accept packages at git revisions.

**Comment by cr6git** [comment]
Back to the latest stable release 1.1.2 (dated 2014-12-08).

File: srcpkgs/remmina/template (hunk 1 above)
Comment (cr6git) [reply srcpkgs/remmina/template]: ok I modified the template.

**Comment by cr6git** [comment]
@the-maldridge why it is still blocked ?

**Comment by the-maldridge** [comment]
Because life is busy. I'll try to get to this tomorrow but maybe @Vaelatern
can get to it faster.

**Pull request closed without merging**